.gitmodules

```diff
@@ -67,3 +67,6 @@
 [submodule "projects/alphacamm"]
 	path = projects/alphacamm
 	url = git@lfna.unizar.es:gifna/alphacamm.git
+[submodule "source/libraries/legacy"]
+	path = source/libraries/legacy
+	url = https://github.com/rest-for-physics/geant4lib.git
```

Comment:
Be careful, this points to geant4lib!

Reply:
> Be careful, this points to geant4lib!

Such a big failure, I don't know how this end up like that. Thanks for pointing to it, now should be fine.